blueprint-test-utils/src/helpers.rs

```diff
@@ -106,8 +104,10 @@ impl BlueprintProcessManager {
             std::path::absolute(&keystore_uri).expect("Failed to resolve keystore URI");
         let keystore_uri_str = format!("file:{}", keystore_uri_normalized.display());
 
-        let in_memory_keystore = FilesystemKeystore::open(keystore_uri_str.clone())
+        let filesystem_keystore = FilesystemKeystore::open(keystore_uri_str.clone())
```

Comment:
If we're using the filesystem keystore, are the keys being written to this when we generate? Are we doing proper cleanup after these tests execute or are developers locally building up a database of junk keys?